x-pack/plugins/maps/public/components/single_field_select.tsx

```diff
@@ -108,14 +106,21 @@ export function SingleFieldSelect({
     }
   }
 
+  const options = fieldsToOptions(fields, isFieldDisabled);
```

Comment:
Need to pass argument `getFieldDisabledReason`

Reply:
```suggestion
  const options = fieldsToOptions(fields, isFieldDisabled, getFieldDisabledReason);
```